aunt-leahs-app/src/constants.jsx

```diff
@@ -104,4 +19,16 @@ export const pages = {
 	VOLUNTEER_SIGN_UP: 'Volunteer Sign Up'
 };
 
-export const requiredText = "Required";
+export const requiredText = 'Required';
+
+export const applicationBaseUrl = 'http://localhost:3000';
+
+export const adminAPIBaseURL = 'https://aunt-leahs-functions-admin.azurewebsites.net';
+export const volunteerAPIBaseURL = 'https://aunt-leahs-functions.azurewebsites.net';
+// export const volunteerAPIBaseURL = 'http://localhost:7071';
```

Comment:
Local admin api base url would use port 7072 so maybe a comment with that will help easily switch between the two